l10n_es_pos/static/src/js/models.js

```diff
@@ -0,0 +1,116 @@
+                result = number;
+            } else {
+                for (var i = 0; i < diff; i++) {
+                    result += "0";
+                }
+                result += number;
+            }
+            return result;
+        },
+        push_simple_invoice: function (order) {
+            if (
+                this.pushed_simple_invoices.indexOf(order.data.simplified_invoice) ===
+                -1
+            ) {
+                this.pushed_simple_invoices.push(order.data.simplified_invoice);
+                ++this.config.l10n_es_simplified_invoice_number;
+            }
+        },
+        _flush_orders: function (orders) {
+            var self = this;
+            // Save pushed orders numbers
+            _.each(orders, function (order) {
+                if (!order.data.to_invoice) {
+                    self.push_simple_invoice(order);
+                }
+            });
+            return pos_super._flush_orders.apply(this, arguments);
+        },
+    });
+
+    var order_super = models.Order.prototype;
+    models.Order = models.Order.extend({
+        get_total_with_tax: function () {
+            var total = order_super.get_total_with_tax.apply(this, arguments);
+            var below_limit = total <= this.pos.config.l10n_es_simplified_invoice_limit;
+            this.is_simplified_invoice =
+                below_limit && this.pos.config.iface_l10n_es_simplified_invoice;
+            return total;
+        },
+        set_simple_inv_number: function () {
+            this.simplified_invoice = this.pos.get_simple_inv_next_number();
+            this.name = this.simplified_invoice;
+            this.is_simplified_invoice = true;
+        },
+        get_base_by_tax: function () {
+            var base_by_tax = {};
+            this.get_orderlines().forEach(function (line) {
+                var tax_detail = line.get_tax_details();
+                var base_price = line.get_price_without_tax();
+                if (tax_detail) {
+                    Object.keys(tax_detail).forEach(function (tax) {
+                        if (Object.keys(base_by_tax).includes(tax)) {
+                            base_by_tax[tax] += base_price;
+                        } else {
+                            base_by_tax[tax] = base_price;
+                        }
+                    });
+                }
+            });
+            return base_by_tax;
+        },
+        init_from_JSON: function (json) {
+            order_super.init_from_JSON.apply(this, arguments);
+            this.to_invoice = json.to_invoice;
+            this.simplified_invoice = json.simplified_invoice;
+        },
+        export_as_JSON: function () {
+            var res = order_super.export_as_JSON.apply(this, arguments);
+            res.to_invoice = this.is_to_invoice();
+            if (!res.to_invoice) {
+                res.simplified_invoice = this.simplified_invoice;
+            }
+            return res;
+        },
```

Comment:
Añadir aquí el `export_for_printing`

        export_for_printing: function () {
            var result = order_super.export_for_printing.apply(this, arguments);
            var company = this.pos.company;
            result.simplified_invoice = this.simplified_invoice;
            result.company.street = company.street
            result.company.zip = company.zip
            result.company.city = company.city
            result.company.state_id = company.state_id

            var base_by_tax = this.get_base_by_tax();
            for (let tax of result.tax_details) {
                tax.base = base_by_tax[tax.tax.id];
            }

            return result;
        },